rust/private/rustc.bzl

```
@@ -174,7 +174,7 @@ def collect_deps(label, deps, proc_macro_deps, aliases, toolchain):
             linker_inputs = dep[CcInfo].linking_context.linker_inputs.to_list()
             libs = [get_preferred_artifact(lib) for li in linker_inputs for lib in li.libraries]
             transitive_noncrate_libs.append(depset(libs))
-            transitive_noncrates.append(depset(dep[CcInfo].linking_context.linker_inputs))
+            transitive_noncrates.append(depset(transitive = [dep[CcInfo].linking_context.linker_inputs]))
```

if it's already a depset, can you just append it directly?

```suggestion
            transitive_noncrates.append(dep[CcInfo].linking_context.linker_inputs)
```

---

Ooh, nice. Yes, done.